README.md

```diff
@@ -1,8 +1,33 @@
 
-# advance-valuation-rulings
+# Advance Valuation Ruling Backend
 
-This is a placeholder README.md for a new repository
+Service providing the public frontend to ARS.
```

Comment:
Perhaps you mean.
`This service provides the backend for the Advance Valuation Ruling Frontend.`